src/services/formatting/smartIndenter.ts

```diff
@@ -38,12 +38,17 @@ namespace ts.formatting {
 
             // no indentation in string \regex\template literals
             const precedingTokenIsLiteral = isStringOrRegularExpressionOrTemplateLiteral(precedingToken.kind);
-            if (precedingTokenIsLiteral && precedingToken.getStart(sourceFile) <= position && precedingToken.end > position) {
+            if (precedingTokenIsLiteral && precedingToken.getStart(sourceFile) <= position && position < precedingToken.end) {
                 return 0;
             }
 
             const lineAtPosition = sourceFile.getLineAndCharacterOfPosition(position).line;
 
+            const indentationOfEnclosingMultiLineComment = getIndentationOfEnclosingMultiLineComment(sourceFile, position, options);

```

Comment:
utilities.ts already exports a brief `isInComment` function. It it worth enclosing this block in an `if` check on that to make it clearer?